### DISCUSSION TITLE
macOS light/dark UI scheme

### DISCUSSION THREAD

**Discussion by ThorstenRhau** [EdenEast/nightfox.nvim#326] [Q&A]
Hello!
Is there a way to configure Nightfox to follow the macOS UI light/dark scheme? Currently my terminal does and it would be great to have this functionality built in to the theme. For example a light=dayfox and dark=nightfox setting would be great.
With kind regards
Thorsten

**Comment by EdenEast** [comment]
Nightfox sets each style to their own `colors/` file (which is used by the `colorscheme` command). You can do this yourself when setting the colorsheme in you config.

```lua
vim.cmd.colorscheme(vim.o.background == "dark" and "nightfox" or "dayfox")
```

**Reply by ThorstenRhau** [reply]
I have a hard time getting this command to work with my nvim. Have tried to switch between light and dark fox themes (from your repo) in the kitty terminal and re-starting cvmi after I switch theme. Each time I launch nvim if defaults to the standard night fox theme.

kitty 0.27.1 created by Kovid Goyal

NVIM v0.8.3
Build type: Release
LuaJIT 2.1.0-beta3
Compiled by brew@Ventura-arm64.local

Features: +acl +iconv +tui
See ":help feature-compile"
